### DISCUSSION TITLE
ideas

### DISCUSSION THREAD

**Discussion by moha-b** [moha-b/moha-b#1] [Q&A] [answered]
looking for ideas for my special repo

**Comment by stray80** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/119838574/208135015-a5d45fc8-d29b-4ced-b427-4f36bf2738e9.png)
![Group 70](https://user-images.githubusercontent.com/119838574/208136274-6bac6962-4b39-4786-a6f2-8c038773a435.png)
